src/pages/tutorials/getting-started-with-spring-boot.md

```diff
@@ -160,11 +152,179 @@ This schema defines one top level field (in the type `Query`):  `bookById` which
+
+The `@SchemaMapping` annotation maps a handler method to a field in the GraphQL schema and declares it to be the `DataFetcher` for that field.
+The field name defaults to the method name, and the type name defaults to the simple class name of the source/parent object injected into the method. In this example, the field defaults to `author` and the type defaults to `Book`.
+The type and field can be specified in the annotation.
+
+For more, see the [documentation for the Spring for GraphQL annotated controller feature](https://docs.spring.io/spring-graphql/docs/current/reference/html/#controllers).
+
+That's all the code we need! Let's run our first query.
+
+## Running our first query
+
+### Enable the GraphiQL Playground
+GraphiQL is a useful visual interface for writing and executing queries, and much more. Enable GraphiQL by adding this config to the `application.properties` file.
+
+```
+spring.graphql.graphiql.enabled=true
+spring.graphql.graphiql.path=/graphiql
+```
+
+### Boot the application
+Start your Spring application. 
+
+Navigate to http://localhost:8080/graphiql or your custom URL.
+
+### Run the query
+Type in the query and hit the play button at the top of the window.
+
+```graphql
+query {
+  bookById(id: "book-1") {
+    id
+    name
+    pageCount
+    author {
+      id
+      firstName
+      lastName
+    }
+  }
+}
+```
+
+You should see a response like this.
+![GraphQL response](/img/graphiQL.png)
+
+We have built a GraphQL server and run our first query. 
+With the help of Spring for GraphQL features, we were able to achieve this with only a few lines of code.
+
+## Initializing GraphQL Java
+Let's dive into further detail into initializing GraphQL Java.
 
-Once we have this file we need to "bring it to life" by reading the file and parsing it and adding code to fetch data for it.
+`GraphQlSource` is the core Spring abstraction for access to the `graphql.GraphQL` instance to use for request execution.
+It provides a builder API to initialize GraphQL Java and build a `GraphQlSource`.
+See the [Spring for GraphQL documentation](https://docs.spring.io/spring-graphql/docs/current/reference/html/#execution-graphqlsource) for more detail on how to use the API to initialize GraphQL Java objects.
 
-We create a new `GraphQLProvider` class in the package `com.graphqljava.tutorial.bookdetails` with an `init` method which will create a `GraphQL` instance:
+To help explain the key concepts, the following examples illustrate how to initialize GraphQL Java *without* Spring for GraphQL features.
+
+To make life a little easier, add [Google Guava](https://github.com/google/guava) to dependencies in `build.gradle`.
+```groovy
+dependencies {
+	implementation 'org.springframework.boot:spring-boot-starter-graphql'
+	implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation 'com.google.guava:guava:26.0-jre' // NEW
+	testImplementation 'org.springframework.boot:spring-boot-starter-test'
```

Comment:
did you want this indentation like this ... perhaps to call it out?

Reply:
No I don't want this, thanks for the pickup!

Reply:
Tab vs spaces strikes again